packages/app/src/app/overmind/state.ts

```diff
@@ -76,6 +80,15 @@ export const state: State = {
   authToken: null,
   error: null,
   user: null,
+  activeAuthorization: derived(
+    ({ user, activeTeam, activeTeamInfo }: State) => {
+      if (!activeTeam || !activeTeamInfo) return 'ADMIN';
+
+      return activeTeamInfo.userAuthorizations.find(
+        auth => auth.userId === user!.id
+      )!.authorization;
```

Comment:
I'm not sure, but I think `user` can be undefined here.

```suggestion
      if (!activeTeam || !activeTeamInfo || !user) return 'ADMIN';

      return activeTeamInfo.userAuthorizations.find(
        auth => auth.userId === user.id
      )!.authorization;
```